.github/workflows/unit-tests.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
     - name: Build C
       run: |
           cd cmd/
-          ./autogen.sh
+          ./autogen.sh --with-unit-tests
```

Comment:
The default is `[with_unit_tests=yes]`. Was there some issue here?

Reply:
no, but since there is no tangible effect of turning it, I'm just making it explicit in case the default is changed

Reply:
Also the tests happen in the `make distcheck`, which will re-call `configure`. So this one is kind of ignored.

Reply:
(That is, it would make more sense in `AM_DISTCHECK_CONFIGURE_FLAGS`)

Reply:
added